pkg/kafka/cronsumer_message.go

```diff
@@ -0,0 +1,87 @@
+package kafka
+
+import "time"
+
+type Header struct {
+	Key   string
+	Value []byte
+}
+
+type Message struct {
+	Topic         string
+	Partition     int
+	Offset        int64
+	HighWaterMark int64
+	Key           []byte
+	Value         []byte
+	Headers       []Header
+	Time          time.Time
+}
+
+type MessageBuilder struct {
```

Comment:
After implementing builder pattern in our `Message` struct, we wanted to extract it new file as `cronsumer_message`. Why we added prefix cronsumer_? It because we want to separate from internal/message.